studio/components/interfaces/Billing/PaymentSummaryPanel/PaymentSummaryPanel.tsx

```diff
@@ -189,7 +202,12 @@ const PaymentSummaryPanel: FC<Props> = ({
                     isChangingPITRDuration ? 'text-scale-1100 line-through' : ''
                   } text-sm`}
                 >
-                  ${(getProductPrice(currentAddons.pitrDuration).unit_amount / 100).toFixed(2)}
+                  $
+                  {(
+                    ((currentSubscription?.addons ?? []).find(
```

Comment:
As we use this pattern multiple times and it's relatively complex, might be worth extracting it to a reusable function